feature-account-impl/src/main/java/io/novafoundation/nova/feature_account_impl/data/sync/RealExternalAccountsSyncService.kt

```diff
@@ -138,12 +139,13 @@ internal class RealExternalAccountsSyncService @Inject constructor(
         .onFailure { Log.d("ExternalAccountsDiscovery", "Failed to sync external accounts", it) }
         .onSuccess { Log.d("ExternalAccountsDiscovery", "Finished syncing external accounts ") }
 
-    private suspend fun notifyAboutAddedAccounts(added: List<AddAccountResult.AccountAdded>) {
+    private suspend fun notifyAboutAddedAccounts(added: List<AddAccountResult.AccountAdded>, deactivatedMetaIds: Set<Long>) {
         added.map { it.toAccountBusEvent() }
             .combineBusEvents()
             ?.let { eventBus.notify(it, source = ACCOUNTS_CHANGED_SOURCE) }
 
-        metaAccountsUpdatesRegistry.addMetaIds(added.map { it.metaId })
+        val updatedAccountsIds = deactivatedMetaIds.toList() + added.map { it.metaId }
+        metaAccountsUpdatesRegistry.addMetaIds(updatedAccountsIds)
```

Comment:
On a separate note: I think we should improve the naming of methods in this class since I thought we only need to add `added` updates there